src/java.base/share/conf/security/java.security

```diff
@@ -28,6 +28,33 @@
 # Properties in this file are typically parsed only once. If any of the
 # properties are modified, applications should be restarted to ensure the
 # changes are properly reflected.
+#
+# The special "include" property can be defined one or multiple times in
+# this file with a filesystem path value. The effect of each definition
+# is to include a referred security properties file inline, adding all
+# its properties. Security properties defined before an include statement
+# may be overridden by properties in the included file, if their names
+# match. Conversely, properties defined after an include statement may
+# override properties in the included file.
+#
+# Included files, as well as files pointed to by java.security.properties,
+# can include other files recursively. Paths may be absolute or relative.
+# Each relative path is resolved against the base file containing its
+# "include" definition, if local. Paths may contain system properties for
+# expansion in the form of ${system.property}. If a system property does
+# not have a value, it expands to the empty string.
```

Comment:
I mentioned this in a previous comment, but if java.security.properties points to an HTTP URL, can it still include a local file with absolute path?

Reply:
Yes, that's intended as said above.